testutils/check-baselines.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+# TODO: We should figure out how to do this as part of GapicGeneratorTest at some point.
```

Comment:
s/We should figure out how to do this/Do this/
s/at some point./so this check can be run locally./

Reply:
Done. I edited slightly to "...so this check can be run with Gradle check command"